setup.py

```diff
@@ -23,5 +23,9 @@
     ],
     install_requires=[
         'wagtail>=1.0.0',
+        'django-contrib-comments>=1.6.1',
+        'django-comments-xtd>=1.5.3',
```

Comment:
Just note that in the readme. I would suspect 1% of people are using that importer.